npm run dev crashes

When starting to develop and running this system I get the following error message:
> maildev@1.0.0-rc2 dev /home/christian/git/MailDev
> node ./scripts/dev.js & npm run css-watch

module.js:328
    throw err;
    ^

Error: Cannot find module '../test/scripts/send.js'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/christian/git/MailDev/scripts/dev.js:2:20)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)

> maildev@1.0.0-rc2 css-watch /home/christian/git/MailDev
> node-sass -wr --output-style compressed -o app/styles assets/styles/style.scss

--- a/scripts/dev.js
+++ b/scripts/dev.js
@@ -1,5 +1,5 @@
 const nodemon = require('nodemon')
-const sendEmails = require('../test/scripts/send.js')
+const sendEmails = require('./send.js')
 
 nodemon({
   script: './bin/maildev',

